### PR TITLE
Navigation pragmatism

### PR DESCRIPTION
**Pull request by Fermain**
After some discussions it's been decided that the nav should only show Course level and down for now. This will change in future.

### CODE REVIEW

**Comment by Fermain** [comment]
Yeah that's pretty much it, I'd like to be more flexible but we are embedding in Teams so it gets a bit confusing going outside your course in the Teams tab

**Comment by Fermain** [comment]
The idea is that you see the Course level navigation no matter how deep you are, and you don't see a nav for pages above

**Comment by Melisa-Zorraindo** [comment]
OK, I assume this would change in the future, the navigation as it looks now for me is hard to use and I have to rely on the back button in the web browser to be able to actually move through content. I've tried to record a video to make it easier to see what I'm seeing, but apparently I don't have that ability in my computer. I'll send some pics

I'm working with sample content. Hackademic should have two programmes, one of it being JavaScript for Front-End Development:
image

JavaScript for Front-End Development should have four courses:
image

But when navigating to it, I cannot navigate further down and I have to use the back button in the browser to continue navigating:
image

I just want to make sure this is what we're looking for with this changes before I approve

**Comment by Fermain** [comment]
My solution to this is pretty hacky and uses magic numbers, so a difference in structure might produce unpredictable results.

**Comment by Fermain** [comment]
A better solution would be to filter the sitemap and exclude anything above a course

**Comment by Melisa-Zorraindo** [comment]
If the programmes using Mollify will all have the same structure, then we can rely on the magic numbers for now and write an issue to enhance navigation once more programmes start using Mollify?

**Comment by Fermain** [comment]
In theory you could have a mollify project that only contains a single lesson - that should be valid. I'm going to have another crack at this after dinner

**Comment by Anclagen** [comment]
I'm not getting any navigation produced at all. 

I'm also getting an error when trying to install all dependencies
```
PS C:\Users\Alexa\Desktop\-mollify-navigation-pragmatism> npx lerna bootstrap
lerna notice cli v6.6.2
lerna ERR! ENAME Package name "@mollify/cli" used in multiple packages:
lerna ERR! ENAME        C:\Users\Alexa\Desktop\-mollify-navigation-pragmatism\packages\cli
lerna ERR! ENAME        C:\Users\Alexa\Desktop\-mollify-navigation-pragmatism\packages\cli
```

**Comment by Anclagen** [comment]
I had to remove the "packages/*"
```
  "workspaces": [
    "packages/types",
    "packages/cli",
    "packages/tts",
    "packages/molly",
    "packages/lms",
    "packages/*"   <---- remove this
  ],
  ```

**Comment by Fermain** [comment]
> You want this patched here? or should I make a bug fix for it? I guess you could probably remove everything else and just leave "packages/*"

The reason that it's like this is to make sure that npm builds them in the correct order if you run `npm run build --workspaces`, and the last one is a catch all for any that didn't get build in a specific order (like docs).

I'd prefer to manually list the build order, as annoying as that is.

File: packages/lms/src/routes/+layout.server.ts [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
+import path from 'path';
 
-export const load: LayoutServerLoad = () => {
-  return {
-    sitemap: getEntityMetaTree('src/routes/content')
-  };
+export const load: LayoutServerLoad = ({ url }) => {
+
+  try {
+    const navPath = path.join('src/routes', path.join(...url.pathname.split('/').splice(1, 3)))
```
Comment (Anclagen) [inline packages/lms/src/routes/+layout.server.ts:8]: This fixes the problem with no navigation showing, its due to the replacement of " " with "%20" in the url.
```suggestion
    const navPath = path.join('src/routes', path.join(...decodeURIComponent(url.pathname).split('/').splice(1, 3)));
```

**Review by Anclagen — CHANGES_REQUESTED** [review]

**Pull request closed without merging**